src/api/note-vdom.js

```diff
@@ -16,7 +16,7 @@ var hasAttribute = require('../utils/vdom/has-attribute');
 var isTag = require('../utils/vdom/is-tag');
 var isEmpty = require('../utils/vdom/is-empty');
 
-var focusOnNote = require('../utils/noting/is-note');
+var focusOnNote = require('../utils/noting/is-note-segment');
```

Comment:
Actually for quite a few of these. But I guess you haven't got around to refactoring these yet?